webapp/.eslintrc.json

```diff
@@ -3,5 +3,8 @@
   "extends": [
     "plugin:@mattermost/react"
   ],
-  "ignorePatterns": ["src/**/*.js"]
+  "ignorePatterns": ["src/**/*.js"],
+  "rules": {
+    "no-console": ["error", {"allow":  ["error"]}]
+  }
```

Comment:
Yes, to allow using `console.error` for printing try-catch errors.

Reply:
yeah, its invaluable having these logs. I remember there were so many instances os saved debugging hours in Focalboard because the user could share their console logs.